worksheet_A/TerminalHacking.cpp

```diff
@@ -7,6 +7,34 @@
 const int wordLength = 5;
 const int numberOfWords = 15;
 
+// Returns number of matching letters
+int getLikeness(std::string guessedWord, std::string secretWord)
+{
+	int result = 0;
+
+	for (int i = 0; i < secretWord.length(); i += 1) {
+
+		if (secretWord.at(i) == guessedWord.at(i)) {
+			result += 1;
+		}
+	}
+
+	return result;
+}
+
+// Checks to see if word is in the list
+bool wordInList(std::string guess, std::set<std::string> options)
```

Comment:
`std::set` has a built-in method for doing this, which also happens to have better time complexity than this (O(log n) vs O(n)) -- see http://stackoverflow.com/questions/1701067/how-to-check-that-an-element-is-in-a-stdset